Maestro/subscriptions.json

```diff
@@ -869,7 +869,7 @@
       "actionArguments": {
         "vsoSourceBranch": "release/2.0.0",
         "vsoBuildParameters": {
-          "ScriptFileName": "build-managed.cmd",
+          "ScriptFileName": "UpdateDependencies.ps1",
           "Arguments": [
             "--",
             "/t:UpdateDependenciesAndSubmitPullRequest",
```

Comment:
The `--` and `/t:UpdateDependenciesAndSubmitPullRequest` lines need to be removed, too. `--` isn't needed in this flow, and the target is already included in the script.